x-pack/test/functional/apps/infra/feature_controls/infrastructure_spaces.ts

```diff
@@ -79,21 +79,19 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
       });
 
       it(`metrics app is inaccessible and Application Not Found message is rendered`, async () => {
-        await PageObjects.common.navigateToApp('infraOps', {
+        await PageObjects.common.navigateToActualUrl('infraOps', '', {
```

Comment:
Nope, this was just an inconsistency on my end...since the server is responding at the endpoint level with a 404, there isn't a benefit to keeping both around.